templates/biography.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title:biography template
+---
+
+## Description
```

Comment:
Is description meant to come before brief summary?  
Which is the shorter of the two?

I think it would help for us to have a short summary *outside* of the table-of-contents structure, and then the longer description can be the first heading, which I see in many other wikis.

See [the pig page on the Minecraft wiki](https://minecraft.wiki/w/Pig).  
"A pig is a passive mob..." would be our "brief summary." It is at the very top and does not belong to any ToC heading.   Then where the "Spawning" header is would be the full "Description," and exist as the first ToC heading.

Reply:
I was thinking that description would be something along the lines of `robovac_catesby is a teapot. he is represented by teapots.` or something jist-y. 
Then brief summary could be like ```While he remained silent and hidden during robovac_v1, later on he would go onto to hold significant part in Ministry of dust``` yadda yadda summary of him. 
Lastly, the appearances bit could maybe a table that just let's people read into events in greater detail than the summary provides.

Reply:
I think I get it-- 
but since the descriptions in the other templates are more long-form, could we move the short description quip above the description section? and add that to the other templates? Does that make sense?

Reply:
The description would be a short description and the brief summary, which I'll drop the brief out of. Is a brief summary of the characters characteristics/actions.

Reply:
I actually have some rough examples in general, let me figure out how to pr these.

Reply:
filed #46 to track this